### PR TITLE
Change the 'id' field to 'userId' on assessor allocations

### PR DESCRIPTION
**Pull request by henricook**
I was working with this case class on a branch and kept getting caught out by 'id' - is it an assessor allocation id? a user id? some other kind of id? So i've changed it to make it clearer

@fayimora @vglushak - i think this is your area of responsibility now :-) do you think i've got everything and it's ok?

### CODE REVIEW

**Comment by vglushak** [comment]
👍 
But for CandidateAllocation it's applicationId, rather then user.

**Comment by fayimora** [comment]
I'm happy with that change. You will need to propagate it to the templates though as they are used there and I dont see any templates in your diff.

**Comment by henricook** [comment]
Exactly why I asked! Templates in admin frontend only I guess right? We use that model for exchange? I'll take a look

**Comment by henricook** [comment]
Oh i see, to @vglushak - the id means a different thing in CandidateAllocation - yipes! Will have a think about it, maybe the base type isn't necessary

**Comment by benjstephenson** [comment]
I would say the base type isn't necessary anymore as the candidate
allocation will change to include all the session related info.

File: app/model/command/Allocation.scala [hunk 1]
```diff
@@ -22,14 +22,13 @@ import model.exchange.AssessorSkill
...
   def status: AllocationStatus
 }
 
 case class AssessorAllocation(
-  id: String,
-  status: AllocationStatus,
-  allocatedAs: AssessorSkill
+                               userId: String,
+                               status: AllocationStatus,
+                               allocatedAs: AssessorSkill
```
Comment (vglushak) [inline app/model/command/Allocation.scala]: https://stackoverflow.com/questions/26880677/intellij-scala-plugins-case-class-indentation-is-absurd

**Comment by vglushak** [comment]
@benjstephenson the way it's organised it does not need to include event/session id (those two parameters passed outside json). We might want to rework it in future, but it works on session level already.
-
LGTM.

**Comment by henricook** [comment]
My solution to this got too ugly, so i'm abandoning. Refactoring the very generic repository with a different ID key was a bit rough.

**Pull request closed without merging**